packages/frontend/src/pages/Files/Files.tsx

```diff
@@ -0,0 +1,13 @@
+import { Typography } from '@mui/material';
```

Comment:
How do you plan to use this component?

Reply:
We would use this page for listing the files of single alfresco folder. Since this does not do anything right now I guess we dont need it in this PR.